ios/Video/DataStructures/VideoSource.swift

```diff
@@ -51,9 +51,9 @@ struct VideoSource {
         } else {
             self.requestHeaders = nil
         }
-        self.startPosition = json["startPosition"] as? Int64
-        self.cropStart = json["cropStart"] as? Int64
-        self.cropEnd = json["cropEnd"] as? Int64
+        self.startPosition = json["startPosition"] as? Float64
+        self.cropStart = (json["cropStart"] as? Float64).flatMap { Int64(round($0)) }
```

Comment:
can you explain why you need .flatMap { Int64(round($0)) } ?
and not in startPosition ?

Reply:
@freeboub 
Thanks for review!

> Looks good, but I really think you need to do the same for android.
Android already works properly with float value.

> can you explain why you need .flatMap { Int64(round($0)) } ?
> and not in startPosition
startPosition used within seek function and seek need `CMTimeMakeWithSeconds` that accept float value only.
however cropStart(cropEnd) is used make CMTimeMake that accept Int value only.
so, I used flatMap for safe type casting (If just use `json["cropStart"] as? Int64` will be return nil when pass float value)

Reply:
OK, thank you, BTW on android we have this code:
int cropStartMs = ReactBridgeUtils.safeGetInt(src, PROP_SRC_CROP_START, -1);
int cropEndMs = ReactBridgeUtils.safeGetInt(src, PROP_SRC_CROP_END, -1);

It can work but it will be 1ms accurate instead of 1microsec.
Do you agree with that ? (not necessary for the PR)

Reply:
@freeboub 
iOS code also works in ms
`rounte($0)` maked rounded value